docs/quick-start/configure-mongodb.txt

```diff
@@ -0,0 +1,56 @@
+.. _laravel-quick-start-connect-to-mongodb:
+
+=================================
+Configure Your MongoDB Connection
+=================================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: test connection, runnable, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Set the Connection String in the Database Configuration
+
+      Open the ``database.php`` file in the ``config`` directory and
+      set the default database connection to ``mongodb`` as shown
+      in the following line:
+
+      .. code-block:: php
+
+         'default' => env('DB_CONNECTION', 'mongodb'),
+
+      Add the following highlighted ``mongodb`` entry to the ``connections`` array
+      in the same file. Replace the ``<connection string>`` placeholder with the
+      connection string that you copied from the :ref:`laravel-quick-start-connection-string`
+      step in the following code example:
```

Comment:
S: not necessary, but could make sense to introduce the code example right after the first sentence ("Add the following highlighted..."), then move the sentence about replacing the placeholders after the snippet.

```suggestion
      in the same file:
      
      // code snippet
      
      Replace the ``<connection string>`` placeholder with the
      connection string that you copied from the :ref:`laravel-quick-start-connection-string`
      step.
```

Reply:
I think both ways can work and there are examples of both in our documentation. I'll keep this as-is for now; open to changing it if there are any strong opinions.